docs/user_guide/assessment.rst

```diff
@@ -1,20 +1,134 @@
+
+Identify the groups that might be harmed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In most applications, we consider demographic groups including historically 
+marginalized groups (e.g., based on gender, race, ethnicity). We should also 
+consider groups that are relevant to a particular use case or deployment context. For example, for 
+speech-to-text transcription, this might include groups who speak a regional dialect or people who are a  
+native or a non-native speaker.
+
+It is also important to consider group intersections, for example, in addition
+to considering groups according to gender and groups according to race, it is 
+also important to consider their intersections (e.g., Black women, Latinx 
+nonbinary people, etc.).
+
+Quantify harms
+^^^^^^^^^^^^^^
+
+Define metrics that quantify harms or benefits:
+
+* In a job screening scenario, we need to quantify the number of candidates that are classified as "negative" (not recommended for the job), but whose true label is "positive" (they are "qualified"). One possible metric is the false negative rate: fraction of qualified candidates that are screened out. Note that before we attempt to classify candidates, we need to determine the construct validity of the "qualified" status; more information on construct validity can be found in :ref:`construct_validity`.
+
+* For a speech-to-text application, the harm could be measured by disparities in the word error rate for different group, measured by the number of mistakes in a transcript divided by the overall number of words.
+
+Note that in some cases, the outcome we seek to measure is not 
+directly available. 
+Occasionally, another variable in our dataset provides a close 
+approximation to the phenomenon we seek to measure. 
+In these cases, we might choose to use that closely related variable, 
+often called a "proxy", to stand in for the missing variable. 
+For example, suppose that in the job screening scenario, 
+we have data on whether the candidate passes the first two stages, 
+but not if they are ultimately recommended for the job. 
+
+As an alternative to the unobservable final recommendation, we could
+therefore measure the harm using the proxy variable indicating whether
+the candidate passes the first stage of the screen.
+If you choose to use a proxy variable to 
+represent the harm, check the proxy variable regularly to ensure it 
+remains useful over time. Our section on :ref:`construct_validity`
+describes how to determine whether a  
+proxy variable measures the intended construct in a meaningful 
+and useful way. It is important to ensure that the proxy is suitable 
+for the social context of the problem you seek to solve. 
+In particular, be careful of falling into one of the :refabstraction_traps. 
+
+Compare quantified harms across the groups
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The workhorse of fairness assessment in Fairlearn are disaggregated metrics, which are 
```

Comment:
Perhaps it's me (non-native speaker here!) but workhorse doesn't sound right to me.
Key piece? Center piece?

Reply:
Changed to centerpiece!